ledger/heavy/replica/jetkeeper.go

```diff
@@ -169,3 +189,12 @@ func (jk *dbJetKeeper) updateSyncPulse(pn insolar.PulseNumber) error {
 	err := jk.db.Set(syncPulseKey(pn), []byte{})
 	return errors.Wrapf(err, "failed to set up new sync pulse")
 }
+
+func (jk *dbJetKeeper) topSyncPulse() insolar.PulseNumber {
```

Comment:
This function is hard to understand.
In discussion with @andreyromancev he suggest to store last synced pulse in constant key and update it

Reply:
+

Reply:
done